Task_8/style.css

```diff
@@ -0,0 +1,373 @@
+/*Игровое поле*/
+.container{
+    margin: 50px auto;
```

Comment:
There is good style rule that (flex properties)
```
display: flex;
justify-content: space-between;
flex-wrap: wrap;
align-items: center;
```

should be in first position for all selectors